std/traits.d

```diff
@@ -3550,6 +3551,176 @@ enum hasMember(T, string name) = __traits(hasMember, T, name);
 }
 
 /**
+ * Whether the symbol represented by the string, member, is a static member of T.
+ *
+ * Params:
+ *     T = Type containing symbol $(D member).
+ *     member = Name of symbol to test that resides in $(D T).
+ *
+ * Returns:
+ *     $(D true) iff $(D member) is static.
+ */
+template isStaticMember(T, string member)
+    if (__traits(hasMember, T, member))
+{
+    import std.meta : Alias;
+    alias sym = Alias!(__traits(getMember, T, member));
```

Comment:
nope, it'll fail if the inspection is made in another module without a public protection.

To be clear anything like that shouldn't be proposed until [something like this](https://github.com/BBasile/DIPs/blob/3d5e3f81c541c6e23c69555a230b4d42a7bb6de6/DIPs/DIP8484.md)  is implemented.

If you merge this without the new behavior against protection for __trait() then you'll get a bug report, certainly. Can't say when, but this will happen.